R/uvhydrograph-render.R

```diff
@@ -43,6 +45,19 @@ uvhydrographPlot <- function(reportObject) {
   
 }
 
+#' Use Secondary Plot
+#' @description Determines where or not a report should use a secondary plot 
+#' @param reportObject the report to be rendered
+#' @return boolean true or false of whether a secondary series is appropriate for report
+useSecondaryPlot <- function(reportObject) {
+  hasReferenceSeries <- hasReferenceSeries(reportObject)
+  hasUpchainSeries <- hasUpchainSeries(reportObject)
+  
+  useSecondaryPlot <- (hasReferenceSeries && !isPrimaryDischarge(reportObject)) || hasUpchainSeries
```

Comment:
For consistency maybe the `isPrimaryDischarge()` call should be pulled out to a variable like `hasReferenceSeries` and `hasUpchainSeries`, or those 2 calls could be embedded into the this statement?